hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/debug/logs/container/utils/ContainerDatanodeDatabase.java

```
@@ -0,0 +1,555 @@
+ */
+public class ContainerDatanodeDatabase {
+  
+  private static String databasePath;
+  private static final int DEFAULT_REPLICATION_FACTOR;
+  
+  static {
+    OzoneConfiguration configuration = new OzoneConfiguration();
+    final String replication = configuration.getTrimmed(
+        OMConfigKeys.OZONE_SERVER_DEFAULT_REPLICATION_KEY,
+        OMConfigKeys.OZONE_SERVER_DEFAULT_REPLICATION_DEFAULT);
+
+    DEFAULT_REPLICATION_FACTOR = Integer.parseInt(replication.toUpperCase());
+  }
+  
+  public static void setDatabasePath(String dbPath) {
+    if (databasePath == null) {
+      databasePath = dbPath;
+    }
+  }
+
+  private static Connection getConnection() throws Exception {
+    if (databasePath == null) {
+      throw new IllegalStateException("Database path not set");
+    }
+    
+    Class.forName(SQLDBConstants.DRIVER);
+
+    SQLiteConfig config = new SQLiteConfig();
+
+    config.setJournalMode(SQLiteConfig.JournalMode.OFF);
+    config.setCacheSize(SQLDBConstants.CACHE_SIZE);
+    config.setLockingMode(SQLiteConfig.LockingMode.EXCLUSIVE);
+    config.setSynchronous(SQLiteConfig.SynchronousMode.OFF);
+    config.setTempStore(SQLiteConfig.TempStore.MEMORY);
+
+    return DriverManager.getConnection(SQLDBConstants.CONNECTION_PREFIX + databasePath, config.toProperties());
+  }
+
+  public void createDatanodeContainerLogTable() throws SQLException {
+    String createTableSQL = SQLDBConstants.CREATE_DATANODE_CONTAINER_LOG_TABLE;
+    try (Connection connection = getConnection();
+         Statement dropStmt = connection.createStatement();
+         Statement createStmt = connection.createStatement()) {
+      dropTable(SQLDBConstants.DATANODE_CONTAINER_LOG_TABLE_NAME, dropStmt);
+      createStmt.execute(createTableSQL);
+      createDatanodeContainerIndex(createStmt);
+    } catch (SQLException e) {
+      System.err.println("Error while creating the table: " + e.getMessage());
```

It would be better to wrap the exception with a custom message, and throw it (instead of printing the error and then throwing the exception again). 
```suggestion
      throw new SQLException("Error while creating the table: " + e.getMessage());
```
As this is now a part of the utils package, any error message printing being handled in the command would be cleaner.